contracts/ronin/random-beacon/RoninRandomBeacon.sol

```diff
@@ -280,7 +271,7 @@ contract RoninRandomBeacon is Initializable, VRF, HasContracts, GlobalConfigCons
     uint256 lastUpdatedPeriod,
     address[] memory cids,
     uint256[] memory trustedWeights
-  ) internal {
+  ) internal onlyActivated(lastUpdatedPeriod) {
```

Comment:
This has been checked `onlyActivated(newPeriod)` in `execWrapUpBeaconPeriod`.
Now checked again with `onlyActivated(lastUpdatedPeriod)`.

Is it intended?

Reply:
this is intended to skip slashing when period changes to `ACTIVATED_AT_PERIOD`